src/modules/vendor/mixins/templateMixin.js

```diff
@@ -30,9 +30,11 @@ export const templateMixin = {
   computed: {
     ...mapState('card', ['card']),
     mediaFileName () {
-      return this.card && this.card.title
-        ? this.card.title.replace(/ /g, '_')
-        : this.cardParent.name.replace(/ /g, '_');
+      if (this.card && this.card.title) return this.card.title.replace(/ /g, '_');
+
+      return get(this.card, 'name')
```

Comment:
get(this.cardParent) ? 
C'est dommage qu'on est donné 'name' pour activity et 'title' pour Questionnaire. Que pensez-vous de creer un ticker pour harmoniser ?

Reply:
je suis d'accord :)